source/renderer/app/components/status/NetworkStatus.js

```diff
@@ -29,7 +20,227 @@ import type { CardanoNodeState } from '../../../../common/types/cardano-node.typ
+    id: 'status.network.dialog.localTimeDifference',
+    defaultMessage: '!!!Local Time Difference',
+    description: 'Local Time Difference',
+  },
+  systemTimeCorrect: {
+    id: 'status.network.dialog.systemTimeCorrect',
+    defaultMessage: '!!!System Time Correct',
+    description: 'System Time Correct',
+  },
+  systemTimeIgnored: {
+    id: 'status.network.dialog.systemTimeIgnored',
+    defaultMessage: '!!!System Time Ignored',
+    description: 'System Time Ignored',
+  },
+  checkingNodeTime: {
+    id: 'status.network.dialog.checkingNodeTime',
+    defaultMessage: '!!!Checking Node Time',
+    description: 'Checking Node Time',
+  },
+  cardanoNodeStatus: {
+    id: 'status.network.dialog.cardanoNodeStatus',
+    defaultMessage: '!!!CARDANO NODE STATUS',
+    description: 'CARDANO NODE STATUS',
+  },
+  cardanoNodeStatusRestarting: {
+    id: 'status.network.dialog.cardanoNodeStatusRestarting',
+    defaultMessage: '!!!Restarting Cardano Node...',
+    description: 'Restarting Cardano Node...',
+  },
+  cardanoNodeStatusRestart: {
+    id: 'status.network.dialog.cardanoNodeStatusRestart',
+    defaultMessage: '!!!Restart Cardano Node',
+    description: 'Restart Cardano Node',
+  },
+  cardanoNodeDiagnostics: {
+    id: 'status.network.dialog.cardanoNodeDiagnostics',
+    defaultMessage: '!!!Cardano Node Diagnostics',
+    description: 'Cardano Node Diagnostics',
+  },
+  realtimeStatisticsMonitor: {
+    id: 'status.network.dialog.realtimeStatisticsMonitor',
+    defaultMessage: '!!!Realtime statistics monitor',
+    description: 'Realtime statistics monitor',
+  },
+  cardanoNodeState: {
+    id: 'status.network.dialog.cardanoNodeState',
+    defaultMessage: '!!!Cardano Node State',
+    description: 'Cardano Node State',
+  },
+  cardanoNodeResponding: {
+    id: 'status.network.dialog.cardanoNodeResponding',
+    defaultMessage: '!!!Node Responding',
+    description: 'Node Responding',
+  },
+  cardanoNodeSubscribed: {
+    id: 'status.network.dialog.cardanoNodeSubscribed',
+    defaultMessage: '!!!Node Subscribed',
+    description: 'Node Subscribed',
+  },
+  cardanoNodeTimeCorrect: {
+    id: 'status.network.dialog.cardanoNodeTimeCorrect',
+    defaultMessage: '!!!Node Time Correct',
+    description: 'Node Time Correct',
+  },
+  cardanoNodeSyncing: {
+    id: 'status.network.dialog.cardanoNodeSyncing',
+    defaultMessage: '!!!Node Syncing',
+    description: 'Node Syncing',
+  },
+  cardanoNodeInSync: {
+    id: 'status.network.dialog.cardanoNodeInSync',
+    defaultMessage: '!!!Node In Sync',
+    description: 'Node In Sync',
+  },
+  localTimeDifferenceChecking: {
+    id: 'status.network.dialog.localTimeDifferenceChecking',
+    defaultMessage: '!!!Checking...',
+    description: 'Checking...',
+  },
+  localTimeDifferenceCheckTime: {
+    id: 'status.network.dialog.localTimeDifferenceCheckTime',
+    defaultMessage: '!!!Check time',
+    description: 'Check time',
+  },
+});
+
 type Props = {
+  systemInfo: Object,
```

Comment:
@DeeJayElly please specify type in more detail 🙏

Reply:
@nikolaglumac done